algoliasearch/src/main/java/com/algolia/search/saas/BaseAPIClient.java

```diff
@@ -62,7 +62,9 @@
     private List<String> readHosts;
     private List<String> writeHosts;
 
-    /** HTTP headers that will be sent with every request. */
+    /**
+     * HTTP headers that will be sent with every request.
+     */
```

Comment:
1. Why not. But you still have to review the reformatting commit to check that's it's only reformatting... Kind of egg-and-chicken problem. ;)
2. Huh... that's diff, not blame?